assets/js/default-calendar.js

```diff
@@ -393,6 +393,30 @@ jQuery(function ($) {
 			calendarBubbles(calendar);
 		});
 	};
+	/*
+	 * Print calendar option code
+	 */
+	$('#print-calendar-button').on('click', function () {
+		var $divToPrint = $('.simcal-calendar').clone(true);
+		$('body').children().hide();
+
+		$('main').css({ display: 'none' });
```

Comment:
Can we instead add a class in the body? Styles can be in CSS directly.

Reply:
I was talking about moving all styling changes to CSS

Reply:
So you mean rather then using `$('body').children().hide();` I can use this `.css({ display: 'none' }`